cpanfile

```diff
@@ -7,6 +7,8 @@ requires 'URI::Escape';
 requires 'Encode';
 requires 'Digest::MD5';
 requires 'AWS::Signature4';
+requires 'Digest::SHA';
```

Comment:
`Digest::SHA` and `JSON::PP` are added to dependencies. Amazon::S3::Thin is expressing "low dependency" but I guess it's okay because these modules are already in core (from perl-5.13.9).